porttime/ptmacosx_mach.c

```diff
@@ -2,14 +2,21 @@
 
 #include <stdlib.h>
 #include <stdio.h>
-#include <CoreAudio/HostTime.h>
 
 #import <mach/mach.h>
 #import <mach/mach_error.h>
 #import <mach/mach_time.h>
 #import <mach/clock.h>
 #include <unistd.h>
 #include <AvailabilityMacros.h>
+#include <MacTypes.h>

```

Comment:
We should probably spell all `UInt64` as `uint64_t` from `stdint.h`. That way we wouldn't need this legacy header from the Carbon API.

Reply:
OK, but why MacTypes.h instead of stdint.h?

Reply:
That's where `UInt64` is defined, which I used for consistency here. I'd be happy to change to `uint64_t` from `stdint.h`, just might be worth updating the other uses of that too.

Reply:
On macOS, some other header seems to include `MacTypes.h` transitively, on iOS apparently not, so we'd have to include it explicitly (unless we also want to migrate all other uses of the capitalized spellings, i.e. `UInt64` etc.)

Reply:
Oh, I get it now. I think it would be good to just globally change UInt64 to uint64_t and remove any includes that are there just for UInt64. I normally hate changing a lot of things, but this seems very safe.